src/main/java/de/htwberlin/madlib/MadLib.java

```diff
@@ -8,7 +10,25 @@
 
   public String create() {
     // TODO: implement this method
+    Random randomNumber = new Random();
+
+    int randomAdjective = randomNumber.nextInt(ADJECTIVES.length);
+    int randomVerb = randomNumber.nextInt(VERBS.length);
+    int otherRandomVerb = randomNumber.nextInt(VERBS.length);
+    int randomFamousPerson = randomNumber.nextInt(VERBS.length);
+
+    return "Java programming is so " + ADJECTIVES[randomAdjective] + "! It makes me so excited all the time because I love to " + VERBS[randomVerb] + ". Stay hydrated and " + VERBS[otherRandomVerb] +" like you are " + FAMOUS_PERSONS[randomFamousPerson]+"!";
+  }
+
+  public static String[] getADJECTIVES() {
+    return ADJECTIVES;
+  }
+
+  public static String[] getVERBS() {
+    return VERBS;
+  }
 
-    return ""; // TODO: this is only here so that the code can be compiled, please replace it with your result
+  public static String[] getFamousPersons() {
+    return FAMOUS_PERSONS;
   }
```

Comment:
Die getter-Methoden auf die static Arrays sind nicht zwingend notwendig. Du nutzt sie z. B. auch gar nicht. 
Man sollte immer nach dem YAGNI (You ain't gonna need id) Konzept programmieren. Wenn du etwas nicht nutzt, dann programmiere es nicht.